platform/mbed_error.c

```diff
@@ -349,6 +349,10 @@ static void print_error_report(mbed_error_ctx *ctx, const char *error_msg)
             mbed_error_printf("MessageQueue: 0x%X, ", ctx->error_value);
             break;
 
+        case MBED_ERROR_CODE_ASSERTION_FAILED:
+            mbed_error_printf("Assertion failed: ", ctx->error_value);
```

Comment:
This still has the stray `ctx->error_value`.

(You should ideally be getting a warning about that, but unfortunately there's no format checking directives on `mbed_error_printf`. I am intending to add some).